src/Mooc/Videos/Application/Update/VideoTitleUpdaterRequest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+
+namespace CodelyTv\Mooc\Videos\Application\Update;
+
+
+class VideoTitleUpdaterRequest
+{
+    private string $videoId;
+    private string $videoTitle;
+
+    public function __construct(string $videoId, string $videoTitle)
```

Comment:
In this conext you can name your variables with:

```suggestion
    public function __construct(string $id, string $title)
```